### PR TITLE
UOE-9585: Add `it`(IntegrationType) field in logger

### PR DESCRIPTION
**Pull request by pm-saurabh-narkhede**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/pubmaticAnalyticsAdapter.js [hunk 1]
```diff
@@ -371,6 +371,12 @@ function getTgid() {
   return 0;
 }
 
+function getIntegrationType() {
+  const s2sBiddersLength = config.getConfig('s2sConfig')['bidders'].length;
```
Comment (pm-priyanka-deshmane) [inline modules/pubmaticAnalyticsAdapter.js]: Can we have a null check here for s2sConfig? Also we can check if bidders is present and is array.

Comment (pm-priyanka-deshmane) [reply modules/pubmaticAnalyticsAdapter.js]: Rest looks good to me

Comment (pm-saurabh-narkhede) [reply modules/pubmaticAnalyticsAdapter.js]: Done

File: modules/pubmaticAnalyticsAdapter.js [hunk 2]
```diff
@@ -429,6 +429,13 @@ function getTgId() {
   return 0;
 }
 
+function getIntegrationType() {
+  let s2sConfig = config.getConfig('s2sConfig');
+  const s2sBiddersLength = s2sConfig && s2sConfig.bidders && isArray(s2sConfig.bidders) ? s2sConfig.bidders.length : 0;
```
Comment (pm-saurabh-narkhede) [inline modules/pubmaticAnalyticsAdapter.js]: remove `isArray` and use null check

Comment (pm-saurabh-narkhede) [reply modules/pubmaticAnalyticsAdapter.js]: Done

**Pull request merged**